tests/e2e/core-tests/core-tests-root.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+ const path = require( 'path' );
+
+ const getCoreTestsRoot = () => {
+     // Figure out where we're installed.
+     // Typically will be in node_modules/, but WooCommerce
+     // uses a local file path (tests/e2e/env).
+     let coreTestsPath = false;
+     const moduleDir = path.dirname( require.resolve( '@woocommerce/e2e-environment' ) );
```

Comment:
This should be `@woocommerce/e2e-core-tests` because we are sure that package exists. We still need to eliminate the `tests/e2e/core-tests` string below. I think it will work if this just returns `moduleDir`.